src/libraries/Microsoft.Win32.Primitives/README.md

```diff
@@ -0,0 +1,15 @@
+# Microsoft.Win32.Primitives
+
+This assembly exposes the types shared among the Win32 APIs like `System.ComponentModel.Win32Exception`.
+
+## Contribution Bar
+
+- [x] [We only consider fixes to maintain or improve quality](../README.md#primary-bar)
+- [x] [We consider PRs that target this library for new source code analyzers](../README.md#secondary-bars)
+- [x] [We consider PRs with refactoring changes due to new language features](../README.md#secondary-bars)
+
+See the [Help Wanted](https://github.com/dotnet/runtime/issues?q=is%3Aopen+is%3Aissue+label%3Aarea-Microsoft.Win32) issues.
+
+## Deployment
+
+`Microsoft.Win32.Primitives` is provided as a [NuGet package](https://www.nuget.org/packages/Microsoft.Win32.Primitives/).
```

Comment:
Incomplete. This is also offered as an inbox package in Microsoft.NETCore.App:
 https://github.com/dotnet/runtime/blob/493574ee470daf8a353203ce7e933f931f986159/src/libraries/NetCoreAppLibrary.props#L33

See this comment, needs to look like this: https://github.com/dotnet/runtime/pull/78121/files#r1018292210